hermes/src/ontology/mod.rs

```diff
@@ -82,3 +107,27 @@ where
     String::deserialize(deserializer)
         .and_then(|string| base64::decode(&string).map_err(|err| Error::custom(err.to_string())))
 }
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LoadMessage {
+    /// The Snips component that was loaded
+    pub component: SnipsComponent,
+    /// Optional id associated to a reload operation of a component
+    pub load_id: Option<String>,
```

Comment:
maybe use `reload_id` or just `id` instead ? `load_id` seems to add a new notion that is not present in the rest of the api